resources/views/messages/checkin.blade.php

```diff
@@ -1 +1,3 @@
-{!! $content['body'] !!}
+<p>{!! $content['body'] !!}</p>
+
+<p><a href="{!! $content['unsubscribe_link'] !!}">Unsubscribe</a></p>
```

Comment:
I'm pretty sure this can be safely escaped! `{{ $content['unsubscribe_link'] }}`